libs/core/langchain_core/prompts/chat.py

```diff
@@ -406,22 +406,27 @@ def get_lc_namespace(cls) -> List[str]:
     def from_template(
         cls: Type[_StringImageMessagePromptTemplateT],
         template: Union[str, List[Union[str, _TextTemplateParam, _ImageTemplateParam]]],
+        *,
         template_format: str = "f-string",
+        partial_variables: Optional[Dict[str, Any]] = None,
         **kwargs: Any,
     ) -> _StringImageMessagePromptTemplateT:
         """Create a class from a string template.
 
         Args:
             template: a template.
             template_format: format of the template.
+            partial_variables: A dictionary of variables that can be used too partially.
             **kwargs: keyword arguments to pass to the constructor.
 
         Returns:
             A new instance of this class.
         """
         if isinstance(template, str):
             prompt: Union[StringPromptTemplate, List] = PromptTemplate.from_template(
-                template, template_format=template_format
+                template,
+                template_format=template_format,
+                partial_variables=partial_variables,
             )
             return cls(prompt=prompt, **kwargs)
         elif isinstance(template, list):
```

Comment:
Could we verify this code path

Reply:
> Could we verify this code path

Sorry, I don't quite understand what you mean.

Reply:
I meant to add unit tests to that that the code path that belongs to the test case when template is a list